v2/fb-xcom-api-validator/src/server/app.ts

```diff
@@ -78,7 +78,7 @@ function schemaErrorFormatter(errors: FastifySchemaValidationError[], dataVar: S
   let erroneousProperty = '';
 
   if (error.instancePath) {
-    erroneousProperty = error.instancePath;
+    erroneousProperty = error.instancePath.replace('/', '');
```

Comment:
Bad idea. `instancePath` might be something like `/asset/assetType`. Just use `/limit` in your tests.